src/gsUtils/gsStopwatch.h

```diff
@@ -72,22 +72,28 @@ template <typename Clock>
 class gsGenericStopwatch
 {
 public:
-    gsGenericStopwatch() { restart(); }
+
+    /// Declares a stop-watch
+    gsGenericStopwatch() : m_value(0) { restart(); }
 
     /// Start taking the time
     void restart() { m_start = Clock::getTime(); }
 
     /// Return elapsed time in seconds.
-    double stop() const { return Clock::getTime() - m_start; }
+    double stop() { return m_value = Clock::getTime() - m_start; }
+
+    /// Returns the last recorded elapsed time
+    double elapsed() const { return m_value; };
 
     friend std::ostream& operator<< (std::ostream& os, const gsGenericStopwatch& sw)
-    { return formatTime(os, sw.stop()); }
+    { return formatTime(os, sw.m_value); }
```

Comment:
I thought of something like:
gsStopWatch watch; // time: 0
... do something:
gsInfo << watch; // time: 0 + do something.
... do something other;
gsInfo << watch; // time : 0 + do something + do something other.
So << would get back the current elapsed time since start.

Reply:
I see, but we usually also need to store the timings..
What you propose will become

 gsStopWatch watch; // time: 0
... do something:
watch.stop();
gsInfo << watch; // time: 0 + do something.
... do something other;
watch.stop();
gsInfo << watch; // time : 0 + do something + do something other.